addon/components/pie-chart.js

```diff
@@ -83,7 +88,7 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         color: d.color,
         label: d.label,
         value: d.value,
-        percent: d3.round(100.0 * d.value / total, this.get('maxDecimalPlace'))
+        percent: 100.0*d.value / total
```

Comment:
I was just checking.  Actually I'm more worried about arbitrary precision than anything else.  Take:
100 * 0.02 / 3 = 0.666666666666667%
We only have 2 significant digits  so this adds arbitrary precision to the number.  Ideally we would simply say that at 2 sig digits we would cut to:
0.66% (no rounding)
or at least that is how a stats person would look at the chart.